source/Cosmos.Build.Installer/Paths.cs

```diff
@@ -85,7 +85,7 @@ public static void UpdateVSPath()
 
         if (xCurrentInstance == null)
         {
-          throw new Exception("The Visual Studio instance is invalid!");
+          throw new Exception("The found Visual Studio instances are not the expected Visual Studio 2017!");
```

Comment:
This exception is thrown when there are more than one VS instances installed, but none of them match the `VSPath` (command line argument). So I think that a better message would be `$"There is more than one Visual Studio instance installed, but none of them matches the specified path '${VSPath}'"`.

Reply:
Just reworded the exception to say the above and committed to the PR branch, this alright?